queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
```

Comment:
- [ ] Make the same change here as the suggestion i made on line `4` above. 

## Requirements 👍 
![image](https://user-images.githubusercontent.com/50990925/235602726-eb62e1c1-d385-4818-9a8b-097751f88de8.png)